doc/source/devices/index.rst

```diff
@@ -44,15 +44,16 @@ Supported Devices
    adi.adis16507
    adi.adl5960
    adi.admv8818
-   adi.adpd188
    adi.adpd1080
+   adi.adpd188
```

Comment:
Is this intended?

Reply:
This was not intended, but the reordering was performed by the automatic tools.